newswires/client/src/SideNav.tsx

```diff
@@ -78,30 +81,6 @@ export const SideNav = () => {
 		[searchHistory],
 	);
 
-	const toggleSupplier = useCallback(
```

Comment:
there's some semi-complex logic that we reuse here, so pulling it into the search context instead